wiki/release/release-process.md

```diff
@@ -21,41 +21,35 @@
 
 # Release Process
 
-
 This page contains instructions for Pulsar committers on how to perform a release.
 
+## Preparation
+
+You should open an discussion in dev@apache.org to notify others that you volunteer to be a release manager. If there are no disagreements, you can start the release process.
+
+For major releases, you should create a new branch named `branch-2.X.0` once all PRs with the 2.X.0 milestone are merged. If some PRs with 2.X.0 milestome are still working in progress and might took much time to complete, you can move it to the next milestone if they are not important. In this case, you'd better to notify the author in the PR.
+
+For minor releases, if there are no disagreements, you should cherry-pick all merged PRs with the `release/X.Y.Z` labels into branch-X.Y. After these PRs are cherry-picked, you should add the `cherry-picked/branch-X.Y` labels.
+
+Sometimes some PRs cannot be cherry-picked cleanly, you might need to create a separated PR and move the `release/X.Y.Z` label from the original PR to it. In this case, you can ask the author to help creating the new PR.
+
+For PRs that are still open, you can choose delaying them to the next release or pinging others to review so that they can be merged.
```

Comment:
```suggestion
For PRs that are still open, you can choose to delay them to the next release or ping others to review so that they can be merged.
```